fakes/mold

```diff
@@ -0,0 +1 @@
+fakes/ld
```

Comment:
Can you try running `fakes/mold --version` and confirm that it works and reports that it's Wild? Just that this symlink target doesn't look right to me